packages/perseus-core/src/widgets/numeric-input/numeric-input-util.ts

```diff
@@ -10,18 +18,36 @@ type NumericInputPublicWidgetOptions = {
     coefficient: PerseusNumericInputWidgetOptions["coefficient"];
     rightAlign?: PerseusNumericInputWidgetOptions["rightAlign"];
     static: PerseusNumericInputWidgetOptions["static"];
-    answerForms?: PerseusNumericInputWidgetOptions["answerForms"];
+    answers: ReadonlyArray<NumericInputAnswerPublicData>;
 };
 
+/**
+ * This data from `answers` is used pre-scoring to give hints
+ * to the learner regarding the format of accepted answers
+ */
+function getNumericInputAnswerPublicData(
+    answer: PerseusNumericInputAnswer,
+): NumericInputAnswerPublicData {
+    const {answerForms, simplify, status} = answer;
+    return {
+        answerForms,
+        simplify,
+        status,
```

Comment:
So although `status` says whether this specific answer is correct, wrong, or ungraded, it doesn't provide enough info to cheat since we don't include the `value` field` from `answer`. Is that correct?